__e2e__/tests/mute-lists.test.ts

```diff
@@ -114,7 +114,7 @@ describe('Mute lists', () => {
 
   it('Shows the mutelist on my profile', async () => {
     await element(by.id('bottomBarProfileBtn')).tap()
-    await element(by.id('selector-2')).tap()
+    await element(by.id('selector-3')).tap()
```

Comment:
I am now seeing the `Media` tab on `selector-2` so updating to 3 here which makes the test pass.